--- !ditz.rubyforge.org,2008-03-06/issue 
title: "OverflowError: repeated string is too long"
desc: Python throws OverflowError when multiplying String with Fixnum results overflow
type: :task
component: core
release: 1.0.0dev1
reporter: Sumi Tomohiko <SumiTomohiko@neko-daisuki.ddo.jp>
status: :closed
disposition: :fixed
creation_time: 2010-08-15 15:45:04.809884 Z
references: []

id: ccffa2c2ee457c675de260cbea2948cf36afa837
log_events: 
- - 2010-08-15 15:45:05.531015 Z
  - Sumi Tomohiko <SumiTomohiko@neko-daisuki.ddo.jp>
  - created
  - ""
- - 2011-07-14 19:41:25.198056 Z
  - Sumi Tomohiko <SumiTomohiko@neko-daisuki.ddo.jp>
  - assigned to release 0.1.0 from unassigned
  - ""
- - 2012-02-22 06:55:31.677440 Z
  - Sumi Tomohiko <SumiTomohiko@neko-daisuki.ddo.jp>
  - changed status from unstarted to in_progress
  - ""
- - 2012-02-22 07:05:49.544734 Z
  - Sumi Tomohiko <SumiTomohiko@neko-daisuki.ddo.jp>
  - closed with disposition fixed
  - ""
